src/airport_gui.cpp

```diff
@@ -495,8 +495,8 @@ class BuildAirportWindow : public PickerWindowBase {
 				break;
 
 			case WID_AP_AIRPORT_LIST: {
-				int num_clicked = this->vscroll->GetScrolledRowFromWidget(pt.y, this, widget, 0, this->line_height);
-				if (num_clicked == INT_MAX) break;
+				int32_t num_clicked = this->vscroll->GetScrolledRowFromWidget(pt.y, this, widget, 0, this->line_height);
+				if (num_clicked == INT32_MAX) break;
```

Comment:
Should these be `Scrollbar::npos`?

Reply:
Yes they should, but also...

> Keep it only to scrollbar for now for this PR for back-portability, and then later it we can update other code that uses it if we decide it needs it?

Technically also replacing `int32_t` with `Scrollbar::size_type`, and everything that depends on that...
Practically making this PR not back-portable any more as it's just changing way too much.

Reply:
Ah yes, fair enough :)